.github/workflows/ci.yml

```
@@ -12,7 +12,8 @@ env:
   CARGO_TERM_COLOR: always
   RUSTFLAGS: -D warnings
   REGISTRY: ghcr.io
-  RUST_VERSION_COV: nightly-2024-06-05
+  RUST_VERSION: 1.80.0
+  NIGHTLY_RUST_VERSION: nightly-2024-07-21
```

```suggestion
  RUST_VERSION: 1.82.0
  NIGHTLY_RUST_VERSION: nightly-2024-10-28
```

Let's bump the rust version as well since we are touching it anyways.

---

addressed: https://github.com/FuelLabs/forc-wallet/pull/220/commits/6b82bc411fae728da93b8f11d3ee393c26625eb3